examples/common/mod.rs

```diff
@@ -1,13 +1,20 @@
-#[cfg(any(
-    feature = "transport-smol",
-    feature = "transport-tokio",
-))]
+use std::{thread, time::Duration};
+
+use systemstat::{saturating_sub_bytes, Platform, System};
+
+#[cfg(any(feature = "transport-smol", feature = "transport-tokio",))]
```

Comment:
Revert these formatting changes (this diff and the next two diffs).

Reply:
Aye aye